### PR TITLE
Updated toDataURL encoderOptions value to be passed as Number

### PR DESCRIPTION
**Pull request by nuxy**
Fix to resolve issue https://github.com/sportheroes/aurelia-image-resizer/issues/11

### CODE REVIEW

**Review by hadrienl — APPROVED** [review]
Thanks :)

File: src/image-resizer/image-resizer.js [hunk 1]
```diff
@@ -211,7 +211,7 @@ export class ImageResizerCustomElement {
           -x, -y, resized[0], resized[1],
           0, 0, this.canvas.width, this.canvas.height);
 
-        this.output = this.canvas.toDataURL(this.type, this.encoderOptions);
+        this.output = this.canvas.toDataURL(this.type, Number(this.encoderOptions));
```
Comment (hadrienl) [inline src/image-resizer/image-resizer.js]: If no `encoderOptions` attribute is passed, value will be 0 : `Number(undefined) === 0`. Please test if value is a string before transforming it to number.

**Review by hadrienl — CHANGES_REQUESTED** [review]

**Comment by nuxy** [comment]
Updated to check for the following:
- `encoder-options=""` is defined (empty `string`)
- `encoder-options` is not defined (value is `null`)

File: src/image-resizer/image-resizer.js [hunk 2]
```diff
@@ -211,7 +211,10 @@ export class ImageResizerCustomElement {
           -x, -y, resized[0], resized[1],
           0, 0, this.canvas.width, this.canvas.height);
 
-        this.output = this.canvas.toDataURL(this.type, Number(this.encoderOptions));
+        let encoderOpts = (this.encoderOptions !== 'string' && this.encoderOptions !== null)
```
Comment (hadrienl) [inline src/image-resizer/image-resizer.js]: Try

```
const encoderOptions = ['string', 'number'].includes(typeof this.encoderOptions)
? Number(this.encoderOptions)
: null;
```

**Review by hadrienl — CHANGES_REQUESTED** [review]

**Review by hadrienl — APPROVED** [review]
👍🏼

**Pull request merged**